src/System Application/App/AI/src/Azure OpenAI/AzureOpenAIImpl.Codeunit.al

```diff
@@ -44,6 +44,7 @@ codeunit 7772 "Azure OpenAI Impl"
         EmptyMetapromptErr: Label 'The metaprompt has not been set, please provide a metaprompt.';
         MetapromptLoadingErr: Label 'Metaprompt not found.';
         EnabledKeyTok: Label 'AOAI-Enabled', Locked = true;
+        FunctionCallingFunctionNotFoundErr: Label 'Function call, %1, not found.', Comment = '%1 is the name of the function';
```

Comment:
Minor: Would  be more in line with the other error if formulated `'Function call not found: %1'`